### PR TITLE
Update version for FlashPrint

### PR DESCRIPTION
**Pull request by kennedyshead**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Review by bevanjkay — APPROVED** [review]

**Comment by kennedyshead** [comment]
Wonder why it tries to download:
`https://en.fss.flashforge.com/10000/software/.zip`
The url should be:
`https://en.fss.flashforge.com/10000/software/2eb66b9f48316f0b089e6b712ece5e82.zip`

**Review by bevanjkay — CHANGES_REQUESTED** [review]
The `pkg` stanza also needs updating to

```suggestion
    pkg "FlashPrint #{version.major}"
```

File: Casks/flashforge-flashprint.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 cask "flashforge-flashprint" do
-  version "4.5.1,1039f4a96ea3c14f9b7c4ef2856653ee"
-  sha256 "2442e58761417cdbf57d4a6f93c1184c7c1f24fde437aa14776008b14098690d"
+  version "1c68fcfadc5c29627057053d5a5af558"
```
[the remark this answers is not in the document]
Comment (kennedyshead) [reply Casks/flashforge-flashprint.rb]: Stupid me! Sorry, fixed now

**Comment by kennedyshead** [comment]
Trying bump command instead ;)

**Pull request closed without merging**